### DISCUSSION TITLE
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "excluded"

### DISCUSSION THREAD

**Discussion by Roman-Tretiakov** [allure-framework/allure2#2704] [Questions & Support] [answered]
Здравствуйте! 
Я использую Allure.Net.Commons и Allure.SpecFlow для формирования Allure отчета о тестировании. Но получаю ошибку Could not read test result file C:\temp\b47fd0ce1b3a4b4da6ce2dcc464583db-result.json для всех тестовых файлов. Тестовый фреймворк написан на C#

**Comment by baev** [comment, marked as answer]
You need to update the Allure-Commandline version to the latest one
